### PR TITLE
added alerts component

### PR DESCRIPTION
**Pull request by anshikaa12**

### CODE REVIEW

**Comment by ashishnnnnn** [comment]
Code seems fine...
One suggestion - try to host this on netlify ..so that we can preview the live changes

File: documentation/documentation.html [hunk 1]
```diff
@@ -750,14 +886,29 @@ <h2 class="main-h2">Example of Icon with badges</h2>
...
+
+
+
+
+
+
+
+
+
+
```
Comment (sanjay-xdr) [inline documentation/documentation.html:903]: try to remove extra spaces

**Review by sanjay-xdr — COMMENTED** [review]
everything looks good just one suggestion is that try to remove extra spaces

**Comment by AmishMishra11** [comment]
code looks good just in the documentation/documentation.html page try to break individual components as 900+ lines of code will give you troubles later on if you try to change anything in the future

**Comment by anshikaa12** [comment]
> code looks good just in the documentation/documentation.html page try to break individual components as 900+ lines of code will give you troubles later on if you try to change anything in the future

actually its a one page scrollable layout so that is why the HTML file is long. will try to find a solution to it thanks.

File: documentation/documentation.html (hunk 1 above)
Comment (anshikaa12) [reply documentation/documentation.html:903]: yess that is just for my simplification. will remove in the end

**Pull request merged**